Sources/PT.PM/PT.PM.csproj

```diff
@@ -33,4 +33,12 @@
     <ProjectReference Include="..\PT.PM.TSqlParseTreeUst\PT.PM.TSqlParseTreeUst.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Antlr4" Version="4.6.5" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Remove="LanguageDetectors\SqlDialectsLexer" />
```

Comment:
I ran into a problem, which was that the file was included on the builder’s list 2 times, so there were problems compiling. All because of the fact that the file is physically located in the folder from which I hook it. It just a hack for msbuild)

Reply:
Ok then.